Traceback (most recent call last):
  File "/home/lw/venv-ngbem/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/lw/venv-ngbem/lib/python3.12/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lw/venv-ngbem/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/lw/venv-ngbem/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/lw/venv-ngbem/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/lw/venv-ngbem/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
D=HypersingularOperator(fesH1, intorder=12, leafsize=40, eta=3., eps=1e-11, method="aca")
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[5], line 1[0m
[0;32m----> 1[0m D[38;5;241m=[39m[43mHypersingularOperator[49m[43m([49m[43mfesH1[49m[43m,[49m[43m [49m[43mintorder[49m[38;5;241;43m=[39;49m[38;5;241;43m12[39;49m[43m,[49m[43m [49m[43mleafsize[49m[38;5;241;43m=[39;49m[38;5;241;43m40[39;49m[43m,[49m[43m [49m[43meta[49m[38;5;241;43m=[39;49m[38;5;241;43m3.[39;49m[43m,[49m[43m [49m[43meps[49m[38;5;241;43m=[39;49m[38;5;241;43m1e-11[39;49m[43m,[49m[43m [49m[43mmethod[49m[38;5;241;43m=[39;49m[38;5;124;43m"[39;49m[38;5;124;43maca[39;49m[38;5;124;43m"[39;49m[43m)[49m

[0;31mTypeError[0m: HypersingularOperator(): incompatible function arguments. The following argument types are supported:
    1. (space: ngcomp::FESpace, definedon: Optional[ngcomp::Region] = None, intorder: int = 3, leafsize: int = 40, eta: float = 2.0, eps: float = 1e-06, method: str = 'aca', testhmatrix: bool = False) -> ngbem._ngbem.IntegralOperator

Invoked with: <ngsolve.comp.H1 object at 0x72be7abc3d70>; kwargs: intorder=12, leafsize=40, eta=3.0, eps=1e-11, method='aca'

